azure.yaml

```diff
@@ -41,7 +41,7 @@
     - name: kubernetes
       version: 1.16.3
     - name: containerlinux
-      version: 2247.6.0
+      version: 2191.5.0
```

Comment:
I made a mistake on the 1.16.3 release for Azure. I changed the `containerlinux` version bundle but didn't actually change the version so I'm reverting it here to match reality.